### PR TITLE
Feature/prc/publish podspec

### PR DESCRIPTION
**Pull request by qalvapps**
updated podspec

### CODE REVIEW

File: Specs/BVLinearGradient.podspec [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+require 'json'
+key = "BVLinearGradient:version"
```
Comment (tuncaulubilge) [inline Specs/BVLinearGradient.podspec]: We can get rid of this one

Comment (qalvapps) [reply Specs/BVLinearGradient.podspec]: ah yes . removed

File: Specs/RNDeviceInfo.podspec [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+require 'json'
+key = "RNDeviceInfo:version"
+version = JSON.parse(File.read('localpodversions.json'))["#{key}"]
+#version = "0.9.3"
```
Comment (tuncaulubilge) [inline Specs/RNDeviceInfo.podspec]: I think we can delete these commented versions

File: Specs/React.podspec [hunk 3]
```diff
@@ -0,0 +1,212 @@
+# coding: utf-8
+require "json"
+
+package = JSON.parse(File.read(File.join( "package.json")))
+version = package['version']
+version = 0.54.2
```
Comment (tuncaulubilge) [inline Specs/React.podspec]: Don't need it hardcoded, we got the package.json :+1:

File: Specs/Folly.podspec [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+require 'json'
+key = "Folly:version"
+version = JSON.parse(File.read('localpodversions.json'))["#{key}"]
+#version = "2016.09.26.00"
+
+print "#{key} #{version}"
+
+Pod::Spec.new do |spec|
+  spec.name = 'Folly'
+  spec.version = '2016.09.26.00'
```
[the remark this answers is not in the document]
Comment (qalvapps) [reply Specs/Folly.podspec]: good spot. done

File: Specs/React.podspec [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+                              "React/Inspector/*"
+  end
+
+  s.subspec "tvOS" do |ss|
+    ss.dependency             "React/Core"
+    ss.source_files         = "React/**/RCTTV*.{h, m}"
+  end
+
+  s.subspec "jschelpers" do |ss|
+    ss.dependency             "Folly", "2016.09.26.00"
```
Comment (tuncaulubilge) [inline Specs/React.podspec]: Do we need the version here for folly? If so, can we also pull it out from package.json

File: Specs/React.podspec [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+    ss.compiler_flags       = folly_compiler_flags
+    ss.source_files         = "ReactCommon/cxxreact/*.{cpp,h}"
+    ss.exclude_files        = "ReactCommon/cxxreact/SampleCxxModule.*"
+    ss.private_header_files = "ReactCommon/cxxreact/*.h"
+    ss.pod_target_xcconfig  = { "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/Folly\"" }
+  end
+
+  s.subspec "ART" do |ss|
+    ss.dependency             "React/Core"
+    #ss.source_files         = "Libraries/ART/**/*.{h,m}"
```
Comment (tuncaulubilge) [inline Specs/React.podspec]: This commented-out line looks better in my opinion, would be more future-proof

File: TimesReactIOS.podspec [hunk 7]
```diff
@@ -8,14 +8,20 @@ yoga_version = "0.54.2"
 
 Pod::Spec.new do |s|
   s.name         = "TimesReactIOS"
-  s.version      = "0.0.2"
+  s.version      = "0.0.1"
```
Comment (tuncaulubilge) [inline TimesReactIOS.podspec]: Might be a good idea to pull this out from packages/ios-app/package.json and keep it in sync with the ios-app package version. Same for the git tag a couple lines below

File: TimesReactIOS.podspec [hunk 8]
```diff
@@ -25,27 +31,18 @@ Pod::Spec.new do |s|
...
-    'node_modules/react-native-svg/RNSVG.podspec',
-    'node_modules/react-native/third-party-podspecs/DoubleConversion.podspec',
-    'node_modules/react-native/third-party-podspecs/Folly.podspec',
-    'node_modules/react-native/third-party-podspecs/glog.podspec'
-  ]
-  podspecs.each do |podspec_path|
-    spec = Pod::Specification.from_file podspec_path
-    s.dependency spec.name, "#{spec.version}"
-  end
+  s.dependency 'DoubleConversion', "1.1.5"
```
Comment (tuncaulubilge) [inline TimesReactIOS.podspec]: We can get these from localpodversions, less pleases to keep in sync :)

File: TimesReactIOS.podspec [hunk 9]
```diff
@@ -25,27 +34,21 @@ Pod::Spec.new do |s|
...
-    'node_modules/react-native/third-party-podspecs/Folly.podspec',
-    'node_modules/react-native/third-party-podspecs/glog.podspec'
-  ]
-  podspecs.each do |podspec_path|
-    spec = Pod::Specification.from_file podspec_path
-    s.dependency spec.name, "#{spec.version}"
-  end
+
+  s.dependency 'yoga', "#{react_native_version}.React"
+  #s.dependency 'DoubleConversion', "1.1.5"
```
Comment (tuncaulubilge) [inline TimesReactIOS.podspec]: Don't need these comments, 🔥

File: Specs/React.podspec (hunk 5 above)
Comment (qalvapps) [reply Specs/React.podspec]: removed react podspec

**Pull request merged**